pyproject.toml

```diff
@@ -11,7 +11,6 @@ python = ">=3.9.0,<3.10.0"
 dpytools = { git = "https://github.com/ONSdigital/dp-python-tools.git", branch="develop" }
 pandas = "^2.2.1"
 unidecode = "^1.3.8"
-xmltodict = "^0.13.0"
```

Comment:
I think the DEs use this: https://github.com/ONSdigital/dp-data-pipelines/blob/f36ea456a4490730399d69ef34878d7cf10d0030/dpypelines/pipeline/shared/transforms/sdmx/compact/v20/prototype/v1.py#L5